gerrithudsontrigger/src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer/remove.jelly

```diff
@@ -11,6 +11,9 @@
         <l:main-panel>
             <form method="post" action="removeConfirm">
                 <j:choose>
+                    <j:when test="${it.isLastServer()}">
+                        Cannot remove the last server!
+                    </j:when>
```

Comment:
Hi, I was wondering why the last Gerrit server should not be removed. Can you explain the reasoning behind it?

Reply:
Oofh, that was a long time ago :)
But IIRC, it is because of legacy code throughout the plugin that expects there to be at least one server configured (because there used to be only one configured). So to be safe and not break too much this safeguard was added.
I don't know if there is anything left of that legacy code anymore though ;)

Reply:
Thanks for the insight. Legacy strikes again. ;) Would be nice to remove it at some point.